preliz/tests/test_scipy.py

```diff
@@ -13,6 +13,7 @@
     Binomial,
     NegativeBinomial,
     Poisson,
+    Exponential,
```

Comment:
I am adding the distributions in two groups first the continuous then the discrete. Within each group the order is alphabetical.